bin/parseDocs.ts

```diff
@@ -1,376 +1,6 @@
-import * as fs from 'fs';
```

Comment:
moved most of the code (wrapped in a function, but not modified) to bin/parseDocs/docParser.ts